python/ray/autoscaler/_private/commands.py

```diff
@@ -821,6 +822,16 @@ def get_or_create_head_node(
         if not no_restart:
             warn_about_bad_start_command(ray_start_commands, no_monitor_on_head)
 
+        if os.getenv("RAY_enable_autoscaler_v2", "0") == "1":
```

Comment:
Use `RAY_enable_autoscaler_v2=1 ray up cluster.yaml` to make the cluster launcher use the autoscaler v2.